jobboard/templates/jobboard/job.html

```diff
@@ -0,0 +1,28 @@
+{% extends 'base.html' %} {% block content %}
+
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-md p-5 text-align">
+            <h3>{{ job_data.title }} at {{ job_data.company }}</h3>
+            <h5>{{ job_data.description }}</h5>
+
+            {% if user.is_authenticated %}
+            {% if is_student %}
+            {% if is_student_applied %}
+            <p>YOU HAVE ALREADY APPLIED FOR THIS JOB</p>
+            {% else %}
+            <button onclick="location.href = 'apply_for_job/{{ job.id }}'" type="submit">APPLY NOW</button>
+            {% endif %}
+            {% endif %}
```

Comment:
IMO, it is too hard to understand what's going on here without indentation...

Reply:
Yeah, your'e right. not sure why it was uploaded like this.. It wasn't that way on my comp. I'll try to push it again as it's on my comp